kickstarts/base.ks.erb

```diff
@@ -91,13 +91,22 @@ $appliance_root/manageiq-setup.sh
 
 <%= render_partial "post/db_init" %>
 
-# make sure we have the device name in all ifcfg-* files
-ls /etc/sysconfig/network-scripts/ifcfg-* | while read FILE
-do
-  # parse the device name from FILE
-  DEVICE=${FILE##*-}
-  grep -q 'DEVICE=' $FILE || echo "DEVICE=$DEVICE" >> $FILE
-done
+# During the image build, dracut sets up ifcfg-en<whatever> for the device.
+# On the appliances we expect eth0, so remove all the ifcfg-en<whatever> device configurations 
+# and write a config for eth0 allowing the 'network' service to come up cleanly.
+rm -f /etc/sysconfig/network-scripts/ifcfg-en*
+
+cat <<EOF > /etc/sysconfig/network-scripts/ifcfg-eth0
+DEVICE=eth0
+BOOTPROTO=dhcp
+ONBOOT=yes
+TYPE=Ethernet
+USERCTL=no
+NM_CONTROLLED=no
+DEFROUTE=yes
+EOF
+
+systemctl enable network
```

Comment:
`NetworkManager` is enabled for downstream appliances (see https://github.com/ManageIQ/manageiq-appliance-build/pull/315). So we need both `network` and `NetworkManager` services?

Reply:
Yes, if you set s static IP address and write the config file, it will come up when you call `ifup eth0`, but it won't come up on the next reboot unless the `network` service is enabled.